test/mode/modeInsert.test.ts

```diff
@@ -172,6 +172,45 @@ suite('Mode Insert', () => {
     end: ['foo|bar'],
   });
 
+  test("Can handle '<C-u>'", async () => {
```

Comment:
Is there a reason you decided to write the test in this style vs the `newTest`? IMO, should be OK to have this in the style of the `newTest` like the ones you have below.

Reply:
I just mimiced the tests above for &lt;C-w&gt;, I'll change it to the `newTest` style. 👍